service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -107,9 +117,9 @@ public EhrExtractStatus updateEhrExtractStatusAccessStructured(GetGpcStructuredT
     }
 
     public EhrExtractStatus updateEhrExtractStatusAccessDocument(GetGpcDocumentTaskDefinition documentTaskDefinition,
-            String documentName,
-            String taskId,
-            String messageId) {
+                                                                 String documentName,
+                                                                 String taskId,
+                                                                 String messageId) {
```

Comment:
I dont believe this is our coding format have you seen this https://gpitbjss.atlassian.net/wiki/spaces/NIA/pages/2108522539/Java+Coding+Standards

intellij formatted at bottom

Reply:
I guess this came from the fact that the first parameter is now in a new line and IntelliJ wanted to align the rest with this one. I'll change it back.

Reply:
done